lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -1284,17 +1301,32 @@ def show_module_set(type, module_set, regex = nil, minrank = nil, opts = nil) #
             print(tbl.to_s)
           end
 
-          def generate_module_table(type, search_term = nil) # :nodoc:
-            Table.new(
-              Table::Style::Default,
-              'Header'     => type,
-              'Prefix'     => "\n",
-              'Postfix'    => "\n",
-              'Columns'    => [ '#', 'Name', 'Disclosure Date', 'Rank', 'Check', 'Description' ],
-              'SearchTerm' => search_term
-            )
+          def generate_module_table(type, search_terms = [], row_filter = nil) # :nodoc:
+              Table.new(
+                Table::Style::Default,
+                'Header'     => type,
+                'Prefix'     => "\n",
+                'Postfix'    => "\n",
+                'Columns'    => [
+                  '#',
+                  'Name',
+                  'Disclosure Date',
+                  'Rank',
+                  'Check',
+                  'Description'
+                ],
+                'SearchTerm' => row_filter,
+                'Formatters' => {
+                  'Rank' => RankFormatter.new,
+                },
+                'Stylers' => {
+                  'Rank' => RankStyler.new,
+                  'Description' => SubstringStyler.new(search_terms, '%bgmag'),
+                  'Name' => SubstringStyler.new(search_terms, '%bgmag'),
```

Comment:
If you rename `SubstringStyler` to `HighlightSubstringStyler`, it might allow you to default the '%bgmag` argument - which I think would help clean the duplication up 🤞